doc/ref_impl/cntt-ri2/chapters/chapter03.md

```diff
@@ -9,22 +9,93 @@
+1. RI2 enables software vendors of CNFs and cloud container platforms to deploy a reference implementation in their development environment to develop and validate against.
+
+As a result of those two main use cases, RI2 must be deployable in different lab environments:
+
+1. internal development labs of software vendors, operators and 3rd party test labs,
+1. public community labs (e.g. OPNFV),
+1. resources hosted by cloud infrastructure providers (e.g. Packet.com)
+
+In addition, lab environments must meet specific requirements on hardware resources (e.g. compute and storage), capabilities (e.g. support for SR-IOV), and connectivity (e.g., number of NICs) to run the RI2. The following sections describe the resulting requirements in detail.
+
+
 
 <a name="3.2"></a>
 ## 3.2 Access and Connectivity
 
-> Describe the access and connectivity requirements for any lab used to deploy an RI.
+This section describes the access and connectivity requirements for labs used to deploy an RI.
+
+
+<a name="3.2.1"></a>
+### 3.2.1 Lab Access
+
+The requirements on and means of accessing a given lab differ per hosting organization and use case:
+
+* **Community labs**, such as those hosted by OPNFV, must provide remote access from the Internet to allow usage of the lab resources by community members. Remote access shall be managed and secured by means of VPN or SSH gateways. Both the public and out-of-band management networks of the lab shall be accessible from the VPN / SSH gateway host.
+
+* Access to lab resources hosted by **public cloud providers** is managed by the cloud provider. If a public cloud provider is chosen to host community-based development resources, access to those resources shall be available to all community members by means of a application process. 
+
+* Access to **internal development and test labs** of software vendors, operators and 3rd party test labs is managed by the corresponding entity and not expected to be available to community members outside of the respective organization.
+
+
+<a name="3.2.2"></a>
+### 3.2.2 Connectivity
+
+The network fabric in the lab must support the following networks:
+
+* Out-of-Band hardware management
+* Admin (host OS access)
+* Public (external / Internet facing network)
+* Private (cluster internal network)
+* Storage
+
+The network topology should provide at least 2 networks with preallocated IP addressing schemes for the out-of-band management network and the Public network. The Public network must be able to reach / access the public Internet, preferably directly or via an enterprise proxy. At least IPv4 addresses must be available, ideally IPv6 address space should also be supported. At least 1 network interface must be capable of performing PXE boot and that network must be available to both the jump / test host and each physical server.
```

Comment:
req.inf.ntw.04 The Architecture must support dual stack IPv4 and IPv6 for Kubernetes workloads.

Thus, both IPv4 and IPv6 support is required

Reply:
done